[DOCS] More doc updates to remove the CLI (#7874)
great-expectations/great_expectations: docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_azure_blob_storage.md

```

### 5. Confirm that the new Expectation Suites have been added

If you followed the optional step to copy your existing Expectation Suites to Azure blob storage, you can confirm that Great Expectations can find them by running the following Python code:

```python
import great_expectations as gx

context = gx.get_context()
context.list_expectation_suite_names()
```

> Your output should include the Expectations you copied to Azure blob. If you did not copy Expectations to the new Store, they will not be listed.

Reviewer: ```suggestion
Your output should include the Expectations you copied to the Azure blob. Expectations that weren't copied to the new are not listed.
```
